metriq_gym/benchmarks/bseq.py

```diff
@@ -192,3 +197,34 @@ def poll_handler(
             largest_connected_size=lcs,
             fraction_connected=lcs / job_data.coloring.num_nodes,
         )
+
+    def upload_handler(
+        self, job_data: BenchmarkData, result_data: BenchmarkResult, dispatch_time: datetime, submission_id: int, platform_id: int,
+    ) -> None:
+        client = MetriqClient(os.environ.get("METRIQ_CLIENT_API_KEY"))
```

Comment:
It would be nice to move things like the creation of a MetriqClient object to an `upload_job` function in run.py, similarly to how we do for dispatching and polling with the device object. Same